.travis.yml

```diff
@@ -38,3 +41,10 @@ script:
 matrix:
   allow_failures:
     - os: osx
+
+install:
+  # libmicrohttpd (the one in ubuntu 12.04 is too old)
```

Comment:
Travis is using 14.04. Is it possible to get the packages from repo directly?

Reply:
The default libmicrohttpd isn't updated https://travis-ci.org/Zilliqa/Zilliqa/jobs/346214265 , so explicitly fetching this version